third_party/tutorial.proto

```diff
@@ -123,9 +123,16 @@ message InfoBox {
 };
 
 message CodeBlock {
-  repeated InlineContent inline_contents = 1;
-  string code_location = 2;
-  string language_hint = 3;
+  // TODO: Use this instead of static highlighting
+  // currently being used to determine
```

Comment:
I think a line got cut off here?

Reply:
as in?? I see it as I have it. I am forcibly line wrapping after some under 60 character limit so that the Diff on here don't line wrap since tabs are shows as ~6 spaces/chars.

Reply:
Bump

Reply:
> Use this instead of static highlighting currently being used to determine

Determine what?

Reply:
Oh I see